inc/core/acf_blocks/sites/cleanbc/project-query.php

```diff
@@ -9,12 +9,11 @@
 
 namespace Bcgov\Theme\Block;
 
+// Create id and class attributes allowing for custom "className" and "align" values.
 $elId = 'project-' . $block['id'];
 if ( ! empty( $block['anchor'] ) ) {
     $elId = $block['anchor'];
 }
-
-// Create class attribute allowing for custom "className" and "align" values.
 $className = 'project';
```

Comment:
mmm, did this pass the phpcs, as this is not a proper php variable name, at least from WordPress's perspective. The context switch between multiple languages can do this :( i also see on line 40 you used the proper naming of $category_detail.

Reply:
Yes, passed all phpcs checks. I'll play around with this as I continue to evolve it but yes, I was jumping back and forth between JS and PHP at the time of writing this, so my context was polluted.